packages/web-pkg/src/composables/piniaStores/extensionRegistry.ts

```diff
@@ -7,11 +7,12 @@ import { Item } from '@ownclouders/web-client/src/helpers'
 import { useConfigStore } from './config'
 import { FolderView } from '../../ui'
 
+export type AvailableExtensions = 'action' | 'search' | 'sidebarNav' | 'sidebarPanel' | 'folderView'
```

Comment:
@dschmidt are you fine with this?

Reply:
If you only allow these as type... you are preventing apps from registering their own extension types 😏

Reply:
Good point, thx!